lib/docker/template/cache.rb

```diff
@@ -17,12 +17,10 @@ def context(builder, context)
         builder.repo.cache_dir.rm_rf
         $stderr.puts Simple::Ansi.yellow(format("Copying context for %s", builder.repo))
         cache_dir = builder.repo.cache_dir
-        cache_dir.parent.mkdir_p
+        cache_dir.mkdir_p
```

Comment:
This was changed to have the [tag] cache folder created first, to copy the README file into it.
In the code, it is copied before the Dockerfile from /tmp